patches/@endo+ses-ava+0.2.27.patch

```diff
@@ -0,0 +1,250 @@
+diff --git a/node_modules/@endo/ses-ava/exported.js b/node_modules/@endo/ses-ava/exported.js
+index f4cba01..e69de29 100644
+--- a/node_modules/@endo/ses-ava/exported.js
++++ b/node_modules/@endo/ses-ava/exported.js
+@@ -1 +0,0 @@
+-import './src/types.js';
+diff --git a/node_modules/@endo/ses-ava/src/ses-ava-test.js b/node_modules/@endo/ses-ava/src/ses-ava-test.js
+index 36038ad..00f4114 100644
+--- a/node_modules/@endo/ses-ava/src/ses-ava-test.js
++++ b/node_modules/@endo/ses-ava/src/ses-ava-test.js
+@@ -1,18 +1,8 @@
+ // @ts-check
+ import 'ses';
+-import './types.js';
+ 
+ const { apply } = Reflect;
+ 
+-/**
+- * Just forwards to global `console.error`.
+- *
+- * @type {Logger}
+- */
+-const defaultLogger = (...args) => {
+-  console.error(...args);
+-};
+-
+ /**
+  * Determine if the argument is a Promise.
+  * (Approximately copied from promiseKit.js)
+@@ -24,21 +14,46 @@ const isPromise = maybePromise =>
+   Promise.resolve(maybePromise) === maybePromise;
+ 
+ /**
+- * @type {LogCallError}
++ * @typedef {(...args: unknown[]) => void} Logger
++ */
++
++/**
++ * Calls `thunk()` passing back approximately its outcome, but first
```

Comment:
It looks like the following terminology problem was already there. But this PR might still be a good opportunity to fix it.

The term `thunk` here seems to refer to the `func` parameter below. If true, they should both use the same name. The CS term `thunk` that I'm familiar with refers only to a function/closure with zero arguments, which `func` is not. So if they are referring to the same thing, `func` is a better choice.

Hmm. I think I see how this terminology problem happened (and I think I may have perpetrated it). The `func` together with the `args` may have previously been packaged together as one `thunk` argument that was invoked with no args. But that always created a yet deeper stack exactly in the place where we're trying to generate good diagnostic info. So by separating them in this way, we're able to make the diagnostic stacks clearer. Yeah, I think I remember doing this. Sorry I did not clean up the comment.

Reply:
If you want to now feel free to submit a suggestion through Github UI or push a commit on the branch.

Reply:
Manually editing a patch seems a bit much, so https://github.com/endojs/endo/pull/1234 instead ;)

Resolving here.